include/flexflow/simulator.h

```diff
@@ -76,6 +84,8 @@ struct CostMetrics {
   // 2. we call Simulator::free_all before measuring an operator
   // Therefore, the current memory usage of an operator is (size_t)sim->offset
   size_t inputs_memory = 0, outputs_memory = 0, weights_memory = 0;
+  ///< Real memory usage of Op* considering parallelization over devices
```

Comment:
What does "Real" mean here? Per-device?

Reply:
Nope, the memory usage among all devices (i.e. need to consider the parallelization strategy)